neoexchange/astrometrics/ephem_subs.py

```diff
@@ -2247,12 +2247,17 @@ def convert_to_ecliptic(equatorial_coords, coord_date):
 
 
 def orbital_pos_from_true_anomaly(nu, body_elements):
-    """Give Cartesian coordinates for a given true anomaly in radians for s given set of orbital elements"""
+    """Give Cartesian coordinates for a given true anomaly in radians for a given set of orbital elements"""
     a = body_elements['meandist']
     e = body_elements['eccentricity']
     i = radians(body_elements['orbinc'])
     om = radians(body_elements['longascnode'])
     w = radians(body_elements['argofperih'])
+    if not a:  # Comet
+        if e < 1:  # Periodic
+            a = body_elements['perihdist'] / (1 - e)
+        else:  # Non-periodic, not going to deal with this right now.
```

Comment:
Think you can use `r = q * (1+e) / (1+e*cos(nu))` to calculate radius vector for objects with _e_=0.98..1.1. Example object: http://127.0.0.1:8000/target/60979/lc/

Reply:
Technically, the given equation is accurate for everything but e=1. But ultimately it's a plotting issue I was having. 
I'm trying to resolve it, but I noticed we were getting more rollbar issues and wanted to get the PR ready.